code/modules/client/preferences.dm

```diff
@@ -1679,33 +1679,114 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						hair_color = sanitize_hexcolor(new_hair, 6)
 
 				if("hair_style")
+					var/list/snowflake_hair_list = list()
```

Comment:
Marking this as an example but these problems are throughout the file. Please remove any unnecessary edits, and mark ALL edits that are kept.

Reply:
this isn't an unnecessary edit